packages/web_benchmarks/lib/src/browser.dart

```diff
@@ -116,31 +115,31 @@ class Chrome {
       workingDirectory: workingDirectory,
     );
 
-    WipConnection debugConnection;
+    WipConnection? debugConnection;
     if (withDebugging) {
       debugConnection =
-          await _connectToChromeDebugPort(chromeProcess, options.debugPort);
+          await _connectToChromeDebugPort(chromeProcess, options.debugPort!);
```

Comment:
This is fragile; the corresponding check is 30 lines away and laundered through a boolean. It would be much better to replace `withDebugging` with a local cache of `options.debugPort` and null checks on that value.